x-pack/legacy/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -93,6 +99,17 @@ export class TaskRunnerFactory {
             executorResult.data,
             executorResult.retry == null ? false : executorResult.retry
           );
+        } else if (executorResult.status === 'ok') {
```

Comment:
I think we're going to have to settle on what the we allow the executor to return. In the event_log PR, I made it a little more obvious that returning null/undefined from an action execution was the same as return `status: 'ok'`, since that seems like the right thing to do (easier to write an action), and is a little more bullet-proof in case we have actions written in JS.

Reply:
Ok, maybe it's best to assume if you don't return `status: 'error'` or didn't throw that it was successful. Which makes sense to me! Made the change here: https://github.com/elastic/kibana/pull/55227/commits/ce26bccf5a46d97436ac077e3a9721f8006a1e5c.